---
layout: default
---


## Next meeting

**2pm, 11 Apr 2019**  
[Room 124](https://maps.unimelb.edu.au/parkville/building/184/124), Level 1,
Building 184, Parkville Campus, University of Melbourne

Albers PK, McVean G (2018).  [Dating genomic variants and shared ancestry in
population-scale sequencing data](https://doi.org/10.1101/416610), *bioRxiv*.


## About the club

This journal club is hosted by [Melbourne Integrative Genomics
(MIG)](http://research.unimelb.edu.au/integrative-genomics) at the University
of Melbourne.  It is currently managed by [Damjan
Vukcevic](http://research.unimelb.edu.au/integrative-genomics/research/statistical-genetics-vukcevic).


## Mailing list

To be notified about future meetings of the club, please
[subscribe](https://lists.unimelb.edu.au/subscribe/statgen) to the mailing
list.  Membership is open to anyone who is interested.

To manage your subscription or find out more about the list, please visit the
list's [website](https://lists.unimelb.edu.au/info/statgen).


## Proposed papers

*Please send suggestions to
[Damjan Vukcevic](mailto:damjan.vukcevic@unimelb.edu.au).
Papers must be related to genetics or genomics and must have substantial
statistical content.*


Speidel L, Forest M, Shi S, Myers S (2019). [A method for genome-wide genealogy
estimation for thousands of samples](https://doi.org/10.1101/550558), bioRxiv.
([software](https://myersgroup.github.io/relate/))

Hou K, Burch KS, Majumdar A, Shi H, N Mancuso N, Wu Y, Sankararaman S, Pasaniuc
B (2019).  [Accurate estimation of SNP-heritability from biobank-scale data
irrespective of genetic architecture](https://doi.org/10.1101/526855),
*bioRxiv*.

Urbut SM, Wang G, Carbonetto P, Stephens M (2019).  [Flexible statistical
methods for estimating and testing effects in genomic studies with multiple
conditions](https://doi.org/10.1038/s41588-018-0268-8), *Nature Genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30478440),
[biorxiv](https://doi.org/10.1101/096552))

Zhang Y, Qi G, Park JH, Chatterjee N (2018).  [Estimation of complex
effect-size distributions using summary-level statistics from genome-wide
association studies across 32 complex
traits](https://doi.org/10.1038/s41588-018-0193-x), *Nature Genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30104760),
[biorxiv](https://doi.org/10.1101/175406))

Karczewski KJ, ..., Neale BM, Daly MJ, MacArthur DG (2019).  [Variation across
141,456 human exomes and genomes reveals the spectrum of loss-of-function
intolerance across human protein-coding genes](https://doi.org/10.1101/531210),
*bioRxiv*.

Zhu Z, ...,  Visscher PM, Wray NR, Yang J (2018).  [Causal associations between
risk factors and common diseases inferred from GWAS summary
data](https://doi.org/10.1038/s41467-017-02317-2), *Nature Communications*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29335400),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5768719/),
[biorxiv](https://doi.org/10.1101/168674))


## Previous papers

**11 Mar 2019.**  Zhang C, Matsen IV FA (2018).  [Generalizing Tree Probability Estimation via Bayesian Networks](https://papers.nips.cc/paper/7418-generalizing-tree-probability-estimation-via-bayesian-networks), NIPS 2018. ([pdf](https://papers.nips.cc/paper/7418-generalizing-tree-probability-estimation-via-bayesian-networks.pdf), [blog](https://matsen.fredhutch.org/general/2018/12/05/sbn.html))

**10 Dec 2018.**  Khera AV, Chaffin M, Aragam KG, Haas ME, Roselli C, Choi SH,
Natarajan P, Lander ES, Lubitz SA, Ellinor PT, Kathiresan S (2018).
[Genome-wide polygenic scores for common diseases identify individuals with
risk equivalent to monogenic
mutations](https://doi.org/10.1038/s41588-018-0183-z), *Nature Genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30104762))

**19 Nov 2018.**  Kelleher J, Wong Y, Albers P, Wohns AW, McVean G (2018).
[Inferring the ancestry of everyone](https://doi.org/10.1101/458067),
*bioRxiv*.

**5 Nov 2018.**  Bradburd GS, Coop GM, Ralph PL (2018).  [Inferring Continuous
and Discrete Population Genetic Structure Across
Space](https://doi.org/10.1534/genetics.118.301333), *Genetics*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30026187),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6116973/),
[biorxiv](https://doi.org/10.1101/189688))

**8 Oct 2018.**  Shi H, Kichaev G, Pasaniuc B (2016).  [Contrasting the Genetic
Architecture of 30 Complex Traits from Summary Association
Data](https://doi.org/10.1016/j.ajhg.2016.05.013), *AJHG*.
([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/27346688),
[pmc](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5005444/),
[biorxiv](https://doi.org/10.1101/035907))

**10 Sep 2018.**  Tran D, Blei DM (2017).  [Implicit Causal Models for
Genome-wide Association Studies](https://arxiv.org/abs/1710.10742), *arXiv*.
([openreview](https://openreview.net/forum?id=SyELrEeAb))

**13 Aug 2018.**  Mak TSH, Porsch RM, Choi SW, Zhou X, Sham PC (2017).
[Polygenic scores via penalized regression on summary
statistics](https://doi.org/10.1002/gepi.22050), *Genetic Epidemiology*.

**18 Jul 2018.**  He Q, Pilosof S, Tiedje KE, Ruybal-Pes√°ntez S, Artzy-Randrup
Y, Baskerville EB, Day KP, Pascual M (2018).  [Networks of genetic similarity
reveal non-neutral processes shape strain structure in *Plasmodium
falciparum*](https://doi.org/10.1038/s41467-018-04219-3), *Nature
Communications*. ([pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29739937),
['behind the
paper'](https://naturemicrobiologycommunity.nature.com/users/108152-qixin-he/posts/32988-why-a-diverse-approach-is-needed-to-tackle-a-hyperdiverse-malaria-antigen-system))

**27 Jun 2018.**  Durvasula A, Sankararaman S (2018).  [Recovering signals of
ghost archaic admixture in the genomes of present-day
Africans](https://doi.org/10.1101/285734), *bioRxiv*.

**28 May 2018.**  Kelleher J, Thornton K, Ashander J, Ralph P (2018).
[Efficient pedigree recording for fast population genetics
simulation](https://doi.org/10.1101/248500), *bioRxiv*.

**9 May 2018.**  Cabreros I, Storey JD (2017).  [A nonparametric estimator of
population structure unifying admixture models and principal components
analysis](https://doi.org/10.1101/240812), *bioRxiv*.

**18 Apr 2018.**  Chan J, Perrone V, Spence JP, Jenkins PA, Mathieson S, Song
YS (2018).  [A Likelihood-Free Inference Framework for Population Genetic Data
using Exchangeable Neural Networks](https://doi.org/10.1101/267211 ),
*bioRxiv*. ([arxiv](https://arxiv.org/abs/1802.06153))

**28 Mar 2018.**  Li YI, Knowles DA, Humphrey J, Barbeira AN, Dickinson SP, Im
HK, Pritchard JK (2018).  [Annotation-free quantification of RNA splicing using
LeafCutter](https://dx.doi.org/10.1038/s41588-017-0004-9), *Nature Genetics*.
([biorxiv](https://doi.org/10.1101/044107))
